.github/workflows/compile.yml

```diff
@@ -26,5 +26,6 @@ jobs:
           ./spcomp -E textstore.sp -o ../plugins/textstore.smx
           ./spcomp -E textstore_defaults.sp -o ../plugins/textstore_defaults.smx
           ./spcomp -E textstore_generic.sp -o ../plugins/textstore_generic.smx
-          ./spcomp textstore_sqlite.sp -o ../plugins/textstore_sqlite.smx
+          ./spcomp -E textstore_sqlite.sp -o ../plugins/textstore_sqlite.smx
+          ./spcomp -E textstore_mysql.sp -o ../plugins/textstore_mysql.smx
```

Comment:
Compiler will not accept `-E` as there is warnings related in the SQL plugins due to important feature being removed.

Reply:
Ah, that explains what the compiler parameter was for.

Reply:
Ok, i think it's added now.